packages/fxa-graphql-api/package.json

```diff
@@ -14,13 +14,13 @@
     "stop": "pm2 stop pm2.config.js",
     "restart": "pm2 restart pm2.config.js",
     "delete": "pm2 delete pm2.config.js",
-    "test-unit": "JEST_JUNIT_OUTPUT_FILE=../../artifacts/tests/$npm_package_name/jest-unit.xml jest --coverage --runInBand --logHeapUsage --forceExit -t '^(?!.*?#integration).*' --ci --reporters=default --reporters=jest-junit",
-    "test-integration": "JEST_JUNIT_OUTPUT_FILE=../../artifacts/tests/$npm_package_name/jest-integration.xml jest --coverage --runInBand --logHeapUsage -t '#integration' --ci --reporters=default --reporters=jest-junit && yarn test-e2e",
-    "test": "jest --runInBand --logHeapUsage && yarn test-e2e",
+    "test-unit": "npm run build && JEST_JUNIT_OUTPUT_FILE=../../artifacts/tests/$npm_package_name/jest-unit.xml jest --coverage --runInBand --logHeapUsage --forceExit -t '^(?!.*?#integration).*' --ci --reporters=default --reporters=jest-junit",
```

Comment:
@dschom I believe you mentioned there's a better way to do this than add `npm run build &&` in front of the script?

Reply:
Yep, we can do this in the `nx.json` config and the `dependsOn` . I'll have PR up that applies this soon (hopefuly today). Feel free to land this change, and I'll be sure it gets taken care of when I land FXA-7246. 

If you are curious how this works see: https://nx.dev/reference/project-configuration#dependson.